js/app.js

```diff
@@ -217,7 +217,7 @@ function trackerInit() {
     if(!map) load();
 }
 
-// if for some reason, applicationCache is not working, load the app after a 3s timeout
+// load the app after a 3s timeout
 var initTimer = setTimeout(trackerInit, 3000);
```

Comment:
Just wondering - what if we just load straight away without a timeout?

Reply:
In that case, the splash screen won't be seen. Do you want to remove it?

Reply:
Oh, I guess we lose our sponsorship logos.... @darksidelemm there's probably little reason to wait 3 seconds at startup, but we'd a better way of doing sponsorship stuff... happy to keep it for the time being but something maybe we should think about?

Reply:
@argilo lets leave it in the for the time being.

Reply:
If you decide to keep the splash screen, a useful improvement would be to let the rest of the app initialize behind it. At present, things like map tiles and telemetry don't even begin loading until the splash screen disappears.